packages/game-app/cypress/integration/logout.spec.ts

```diff
@@ -0,0 +1,33 @@
+/// <reference types="Cypress" />
+/// <reference types="../support" />
+
+// @ts-ignore
+import {generateRandomCredentials} from "./utils/generators";
+
+context("Logout", () => {
+
+  beforeEach(() => {
+    cy.clearLocalStorage()
+    cy.clearIndexedDB();
+    cy.visit('/login');
+  });
+
+  it("should logout from the email verification required page", () => {
+    const {email, password} = generateRandomCredentials()
+    cy.initializeUser({email: email, password: password}).then(user => {
+      cy.fastLogin(email, password);
+      cy.containsTranslationOf('auth.logout').click();
+      cy.location('pathname').should('equal', '/signup');
+    });
+  })
+
+  it("should logout from the dashboard", () => {
+    const {email, password} = generateRandomCredentials()
+    cy.initializeUser({email: email, password: password, emailVerified: true}).then(user => {
+      cy.fastLogin(email, password);
```

Comment:
Should we actually also test that the user is redirected to the dashboard since it has a verified email?
Or do we prefer keeping these tests fully decoupled?

Reply:
yes, the redirect has been tested in the dedicated one. this should be only related to the logout